gradle-conjure/src/test/groovy/com/palantir/gradle/conjure/ConjurePluginTest.groovy

```diff
@@ -164,29 +167,29 @@ class ConjurePluginTest extends IntegrationSpec {
         updateSettings(prefix)
 
         when:
-        ExecutionResult result = runTasksSuccessfully('check')
-        ExecutionResult result2 = runTasksSuccessfully('check')
+        BuildResult result = runTasksWithConfigurationCache('check')
+        BuildResult result2 = runTasksWithConfigurationCache('check')
 
         then:
-        result.wasExecuted(':extractConjureJava')
-        result.wasExecuted(prefixProject(prefix, 'api-objects:compileJava'))
-        result.wasExecuted(':api:compileConjureObjects')
-        result.wasExecuted(prefixProject(prefix, 'api-jersey:compileJava'))
-        result.wasExecuted(':api:compileConjureJersey')
-        result.wasExecuted(prefixProject(prefix, 'api-undertow:compileJava'))
-        result.wasExecuted(':api:compileConjureUndertow')
-        result.wasExecuted(prefixProject(prefix, 'api-dialogue:compileJava'))
-        result.wasExecuted(':api:compileConjureDialogue')
-
-        result2.wasUpToDate(':extractConjureJava')
-        result2.wasUpToDate(prefixProject(prefix, 'api-objects:compileJava'))
-        result2.wasUpToDate(':api:compileConjureObjects')
-        result2.wasUpToDate(prefixProject(prefix, 'api-jersey:compileJava'))
-        result2.wasUpToDate(':api:compileConjureJersey')
-        result2.wasUpToDate(prefixProject(prefix, 'api-undertow:compileJava'))
-        result2.wasUpToDate(':api:compileConjureUndertow')
-        result2.wasUpToDate(prefixProject(prefix, 'api-dialogue:compileJava'))
-        result2.wasUpToDate(':api:compileConjureDialogue')
+        result.task(':extractConjureJava').outcome == TaskOutcome.SUCCESS
+        result.task(':' + prefixProject(prefix, 'api-objects:compileJava')).outcome == TaskOutcome.SUCCESS
```

Comment:
this is kinda grim from an api perspective, any new test will need to remember to add a `:`. Have `prefixProject` do the right thing and tack on a `:` at the beginning